app/components/post_textbox/post_textbox.js

```diff
@@ -132,7 +131,7 @@ class PostTextbox extends PureComponent {
     };
 
     handleSendMessage = () => {
-        if (!this.state.canSend) {
+        if (!this.canSend()) {
```

Comment:
This isn't actually necessary since the button that calls `handleSendMessage` is only available when `canSend` returns true, but it doesn't hurt to have